packages/amplication-client/src/Entity/SchemaFields.tsx

```diff
@@ -23,24 +22,26 @@ export const SchemaFields = ({
     throw new Error(`Unexpected type ${schema.type}`);
   }
 
-  return (
-    <div>
-      {Object.entries(schema.properties).map(([name, property]) => {
-        if (!property) {
-          throw new Error(`Missing property: ${name}`);
-        }
-        return (
-          <div key={name}>
+  if (schema?.properties) {
+    return (
+      <div>
+        {Object.entries(schema.properties).map(([name, property]) => {
+          if (!property) {
+            throw new Error(`Missing property: ${name}`);
+          }
+          return (
```

Comment:
Is this change related to the current PR? If not, I suggest removing it.
If it is needed, please open a new issue and make the change separately.